frontend/src/utils/colorGenerator.ts

```diff
@@ -0,0 +1,7 @@
+export function generateRandomLightColor (): string {
```

Comment:
nit: let's add something like "hex" in this fn name to make it more distinct from if this was generating, say RGBA values

`generateRandomLightColorHex`